cachito/workers/pkg_managers/general.py

```diff
@@ -119,7 +119,6 @@ def verify_checksum(file_path: str, checksum_info: ChecksumInfo, chunk_size: int
         hasher = hash_file(file_path, chunk_size, checksum_info.algorithm)
     except UnknownHashAlgorithm as exc:
         msg = f"Cannot perform checksum on the file {filename}, {exc}"
-        log.exception(msg)
```

Comment:
> since it'll be logged properly in the worker process

Actually we catch it with `except CachitoError as e:`. So in that case we need to add the *UnknownHashAlgorithm* to expected errors, and the error message will be different. If you ok with that, I can change.

Reply:
I meant to not catch it at all, since this is probably a fatal error. But anyways, I think this could be done during the refactoring of the errors that you are designing.